src/components/app/components/Navbar.js

```diff
@@ -1,9 +1,49 @@
 import React from 'react';
+import { Divider, Icon, Avatar, Layout, Menu } from 'antd';
+import Logo from '../../../images/Logo.png';
+import User from '../../../images/Profile_Picture.png';
 
-import { NAV_TEST } from '../constants';
+import { CLASSNAMES, ICONS, STRINGS } from '../constants';
+
+const { HORIZONTAL, NAME, NEONATAR, OVERVIEW, VERTICAL } = STRINGS;
+const { BARS, BELL, SEARCH } = ICONS;
+const { LEFT_NAV_MENU, HEADER_MENU, RIGHT_NAV_MENU } = CLASSNAMES;
+
+const { Item } = Menu;
+const { Header } = Layout;
 
 const Navabar = () => (
-    <h1>{NAV_TEST}</h1>
+    <Header className={HEADER_MENU}>
+        <Menu mode={HORIZONTAL} className={LEFT_NAV_MENU}>
+            <Item>
+                <Avatar src={Logo} />
+                {NEONATAR}
+            </Item>
+            <Item>
+                <Icon type={BARS} />
+            </Item>
+            <Item>
+                {OVERVIEW}
+            </Item>
+        </Menu>
+        <Menu mode={HORIZONTAL} className={RIGHT_NAV_MENU}>
+            <Item>
+                <Icon type={SEARCH} />
+            </Item>
+            <Item>
+                <Icon type={BELL} />
+            </Item>
+            <Item>
+                <Divider type={VERTICAL} />
+            </Item>
+            <Item>
+                {NAME}
+            </Item>
+            <Item>
+                <Avatar src={User} />
+            </Item>
+        </Menu>
```

Comment:
use array map to creates these menus...you now this, dont repeat yourself

Reply:
Yeaah... On that I have `Icon`, `Avatar` plain text, that will introduce unwanted design.